examples/blockchain.nrm

```diff
@@ -0,0 +1,72 @@
+@error
+@http-status(code="400")
+union blockchain-error
+    = block-validation-error (text reason)
+    | transaction-validation-error (text reason)
+    ;
+
+service blockchain-service (
+    # This is a [blockchain] service example.
```

Comment:
it has to be `[blockchain][]` to make link.

Reply:
:`(

Reply:
@admire93 Nowadays `[blockchain]` also works! Because CommonMark added new feature named [shortcut reference link](http://spec.commonmark.org/0.28/#shortcut-reference-link).